### DISCUSSION TITLE
No consigo visualizar la lista EPC

### DISCUSSION THREAD

**Discussion by Vicanpor** [LITUATUI/M3UPT#46] [General]
Hola Buenas tardes
enhorabuena por este proyecto. 

No consigo visualizar la lista de EPG en archivo de .TXT me sale muy raro algo así: 

9ºŸ¤ÑrÞÇÉ|FÑ–äPò£3P_ìxƒ<,øu~õ~ú^º ÁÐ”	£*†¢"³UA?¿Ø–L©¼c
AÖÁ®ÔRxsÚXò¯ºóº+d<ƒr7ºó
ÿÉŒO™{-·ÆÕ™]é±îOfp	¥›¯6Ë¦LÄ[#2ë¥x5/*S>½Î™¤

necesito el tvg-id="" tvg-name= ¿me puede ayudar?

**Comment by openseabrus** [comment]
What .txt file are you trying to access? There is no .txt file in this repo 🤔

**Comment by Vicanpor** [comment]
este:

https://github.com/LITUATUI/M3UPT/blob/main/EPG/epg-vodafone-pt.xml.gz

**Reply by openseabrus** [reply]
Esse ficheiro é um gzip, tens que extrair o conteúdo primeiro (usando o 7zip ou outro software semelhante). Tem lá dentro um ficheiro .xml com o EPG todo bem formatado. Por exemplo: 
![image](https://github.com/LITUATUI/M3UPT/assets/20615756/acdc897c-0c11-4fa3-b291-3c8cb97a32a7)

**Reply by fcprush** [reply]
Pode.me ensinar a inserir a epg no programa por favor??? Não estou a conseguir fazer isso 😓

**Comment by Vicanpor** [comment]
Es imposible abrir cualquier archivo de EPG. solo si puede bajar y abrir como documento de texto.  y sale en chino coreano y japonés o yo que se....

**Comment by Vicanpor** [comment]
Mil Gracias Ahora si .

gracias

**Comment by fcprush** [comment]
Olá alguém me pode ensinar como utilizar as listas epg por favor??? 
O código de restauro o Ott não funciona no meu Ott 🙁
